vuepress/docs/next/docs/consume/entando-apis.md

```diff
@@ -3,139 +3,80 @@ sidebarDepth: 2
-       - Set `-Dkeycloak.auth.url` to your Keycloak endpoint (including `/auth`), e.g. `-Dkeycloak.auth.url=http://my-keycloak-server/auth`   
-       - Set `-Dkeycloak.client.id` to your client id, e.g. `-Dkeycloak.client.id=quickstart-server`
-       - Set `-Dkeycloak.client.secret` to your client secret, e.g. `-Dkeycloak.client.secret=my-secret`. See [How to find your client secret](#how-to-find-your-client-secret) above.
-    - (Optional) Set`-Djetty.port=8085` if the default port 8080 is already in use. 
-    - (Optional) To skip the docker steps (or if you don't have docker installed/running), add `-DskipDocker=true`
-    
-    Here's a full example:
-    
-        mvn clean package jetty:run-war -Pjetty-local -Pderby -Pkeycloak -Dspring.profiles.active=swagger -Djetty.port=8085 -Dorg.slf4j.simpleLogger.log.org.eclipse.jetty.annotations.AnnotationParser=error -Dkeycloak.auth.url=http://my-keycloak-host/auth -Dkeycloak.client.id=quickstart-server -Dkeycloak.client.secret=my-client-secret -DskipDocker=true
-
-3.  Wait for the application to start.
-
-        [INFO] Started ServerConnector@1355c8be{HTTP/1.1, (http/1.1)}{0.0.0.0:8085}
-        [INFO] Started @66257ms
-        [INFO] Started Jetty Server
-
-4. Navigate to the Swagger UI in a browser at `/entando-de-app/api/swagger-ui.html`
-
-        http://localhost:[your port]/entando-de-app/api/swagger-ui.html
-    
-
-## APIs Overview
+4. To enable the Swagger UI, add the "SPRING_PROFILES_ACTIVE" environment variable. If the variable is already present, add `swagger` to its comma-delimited value list:
+```
+        - name: SPRING_PROFILES_ACTIVE
+          value: default,swagger
+```
 
-The Entando core exposes REST APIs for every action that can be taken in
-the App Builder environment. For example, you can use
-these APIs to create pages, create page templates or to add widgets to
-pages. The APIs can be used to support automation, testing, or
-integrations with external systems.
+5. (Optional) Reset the deployment `spec.replicas` back to 1 if it was scaled to "0" in the previous steps. Save the deployment to update.
 
-### API structure
+### Disable Swagger UI
 
-All of the APIs share a common top level structure. Each response will
-contain a top level entry for `errors`, `metadata`, and `payload`.
+Repeat the steps above, but in step 4, remove `swagger` from the value list.
 
-The `errors` will always contain code and a message string indicating an
-error condition in the request. The `metadata` section is used for
-paging, sorting, filtering and data that is distinct from the body. The
-body of each response is included in the `payload` section of the
-response and varies according to each API.
+## Find Your Client Secret
+You'll need your client credentials to execute the Entando APIs. 
 
-### Models
+1. Login into your Keycloak Administration Console. To find the Kubernetes credentials, see the [Entando Identity Management System](./identity-management.md) page.
```

Comment:
@jyunmitch good catch, no, it's just the admin credentials which can be found in the corresponding k8s secret. "To find the Kubernetes credentials" -> "To find the admin credentials for Keycloak"

Reply:
@nshaw, was trying to avoid "admin" credentials because that is used below in the Swagger authorization steps below (Access APIs on Swagger #4) for the Keycloak admin user (admin/adminadmin) and I wanted to try to distinguish the two

Reply:
@jyunmitch Hmm. "Keycloak admin user" != "admin/adminadmin".  "admin" is the Entando Application admin user.  "entando_keycloak_admin" is the Keycloak admin user. Neither of them are Kubernetes users or credentials.  Not sure that helps.  :(